Alloy/Encoders/TextureCopy.swift

```diff
@@ -36,6 +29,46 @@ final public class TextureCopy {
 
     // MARK: - Encode
 
+    public func callAsFunction(sourceTexture: MTLTexture,
+                               destinationTexture: MTLTexture,
+                               in commandBuffer: MTLCommandBuffer) {
+        self.encode(sourceTexture: sourceTexture,
+                    destinationTexture: destinationTexture,
+                    in: commandBuffer)
+    }
+
+    public func callAsFunction(sourceTexture: MTLTexture,
+                               destinationTexture: MTLTexture,
+                               using encoder: MTLComputeCommandEncoder) {
+        self.encode(sourceTexture: sourceTexture,
+                    destinationTexture: destinationTexture,
+                    using: encoder)
+    }
+
+    public func callAsFunction(region sourceTexureRegion: MTLRegion,
```

Comment:
these and the next one lose semantic with functors, don't you think? unless the object is named `copy`, I believe